.devcontainer/scripts/coderabbit-pr-parser.py

```
@@ -0,0 +1,163 @@
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+    owner, name = REPO.split("/")
+    cmd = ["gh", "api", "graphql", "-F", f"owner={owner}", "-F", f"name={name}", "-F", f"number={int(pr_num)}", "-f", f"query={query}"]
+
+    try:
+        result = subprocess.run(cmd, capture_output=True, text=True, check=True)
```

Ruff (0.14.11)

`subprocess` call: check for execution of untrusted input

(S603)